buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/project-groups.kt

```diff
@@ -59,7 +59,7 @@ object ProjectGroups {
             rootProject.project(":resources"))
 
     val Project.projectsRequiringJava8
-        get() = setOf(rootProject.project(":testingJunitPlatform"))
+        get() = setOf(rootProject.project(":testingJunitPlatform"), rootProject.project("internalPerformanceTesting"))
```

Comment:
I guess we should bump the `performance` subproject to Java 8, too. Same goes for the build scan performance tests, right?

Reply:
I would make them Java 8 on demand, is it necessary for us to bump other projects we're not changing right now?

Reply:
I think we need to bump every project which depends on `internalPerformanceTesting`, given that this project now will produce Java 8 byte code.

Reply:
Instead of adding projects here, we should bump the `INTERAL` project type to Java 8. This is blocked on the Groovy 2.5 update because of some Groovy compile errors though.